workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -21,13 +22,52 @@ export function DashboardPage() {
 		initialDashboardNavigationItems
 	);
 
+	const formatDate = (date: string) => {
+		var dateObject = new Date(date);
+
+		var monthNames = [
+			"Jan", "Feb", "Mar",
+			"Apr", "May", "Jun", "Jul",
+			"Aug", "Sep", "Oct",
+			"Nov", "Dec"
+		];
+
+		var day = dateObject.getDate();
+		var monthIndex = dateObject.getMonth();
+		var year = dateObject.getFullYear();
+
+  		return monthNames[monthIndex] + ', ' + day + ' ' + year;
```

Comment:
The comma should come after the day, not month.

Reply:
It looks like [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) is well supported.  It might have what you need.  This would be nice if we need to support different locales.  We can use the user's preferred language.

Reply:
<img width="190" alt="Screenshot 2023-03-03 at 12 18 21 PM" src="https://user-images.githubusercontent.com/35710155/222797391-87b46423-4ab2-47e2-ab2e-b1a36a35bee7.png">
What is number below?